src/postformatting/index.js

```diff
@@ -43,6 +42,4 @@ export default function cleanup ({
   // remove empty and unwanted rules and at-rules
   finalRuleRemover(ast)
   debuglog('finalRuleRemover')
-
-  return ast
```

Comment:
so the ast is modified (mutated) and hence does not need to be returned; is that it?

Reply:
Yep, that's right